src/pages/en/guides/imports.md

```diff
@@ -17,9 +17,37 @@ Astro uses Vite as its internal build system. Vite provides Astro with built-in
 - Markdown (`.md`)
 - WASM (`.wasm`)
 
-Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. The following applies to files in your `src/` directory, which Astro is ultimately responsible for.
+## `public/`
+Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. 
 
-## JavaScript & ESM
+Reference them in your Astro component template via absolute URL paths, from the `public/` directory.
+
+```
+├── public/
+│   ├── images/
+|   |   └── photo1.jpg
+│   ├── styles/
+|   |   └── global.css
+|   └── utils.js
+```
+
+```astro
+---
+// Astro component script
+---
+<head><link rel="stylesheet" href="/styles/global.css"></head>
+<body>
+  <img src="/images/photo1.jpg"> 
+
+  <script type="module" src="/nav-utils.js"></script>
+</body>
+
+```
+
+## `src/`
```

Comment:
I don't think that this change should force all the existing headers to be nested. I still think the main organization of this page is grouped "by file type".

Can we keep the rest of this page as-is, and instead rename the `public/` header to something like `public/ vs. src/` or `Static Assets in public/`. 

And then rephrase the section/intro a bit to highlight that the rest of the page is about ESM imports, which only work in the `src/` directory.

Reply:
Made a tweak to see what you think: I've put public at the bottom now, emphasizing that the page is mostly about src files, but with a quick link to the bottom.

Reply:
I took a pass and rewrote a bit more. Kept the `public/` reference and moved it higher onto the page, but also made it shorter / less highlighted. I think it works best as a "hey FYI public exists" mention but not a full-fledged "Here's how public works" given that we've already covered that in other parts of the site.

Reply:
No problem! Sorry I kept getting the intent of this page wrong. I was motivated by having a clear `publc/` vs `/src` comparison, as we do see this come up in support, so I thought this might be a page to do that. 

But, I think this can make for its own content (eventually) somewhere else. My `public/` blurb will just go off and join the others in the Land of Misfit Examples for future, more tutorial-like material.  :)

Fixed a pre-existing typo, and otherwise, page looks great!

Reply:
lets see if this improves the support issues that you see, happy to take another look if it becomes an issue.

My hope is that this makes that point clear, but another thing that we could do is to add some bold to the paragraph, it make it pop more:

> Remember that you can also place any static asset in the [`public/` directory](/en/core-concepts/project-structure/#public) of your project, and Astro will copy the directly into your final build. `public/` files are not built or bundled by Astro, which means that any type of file is supported. You can reference a `public/` file by a URL path directly in your HTML templates.